helm/adelphi/templates/spark-image-builder.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       args:
       - --dockerfile=/workspace/kubernetes/dockerfiles/spark/Dockerfile
       - --context=dir:///workspace
-      - {{`--destination={{inputs.parameters.registry_ip}}:30000/adelphi/spark:latest`}}
+      - {{`--destination={{inputs.parameters.registry_ip}}:30000/adelphi/spark`}}
```

Comment:
I removed it while debugging something I thought was related to not finding the Spark image in the custom registry (it wasn't). But I decided to leave it off because it's not really the latest Spark version, it's whatever we pin to elsewhere, so I thought it could be misleading.